src/cli/cmds/audit.js

```diff
@@ -0,0 +1,247 @@
+const fs = require('fs/promises');
+const path = require('path');
+const {
+  files: {loadConfig},
+} = require('@sandworm/utils');
+const {getReport} = require('../..');
+const onProgress = require('../progress');
+const {getIssueCounts, failIfRequested} = require('../utils');
+const summary = require('../summary');
+const logger = require('../logger');
+const checkUpdates = require('../checkUpdates');
+const {outputFilenames} = require('../../files');
+
+exports.command = ['audit', '*'];
+exports.desc = "Security & License Compliance For Your App's Dependencies 🪱";
+exports.builder = (yargs) =>
+  yargs
+    .option('o', {
+      alias: 'output-path',
+      demandOption: false,
+      default: 'sandworm',
+      describe: 'The path of the output directory, relative to the application path',
+      type: 'string',
+    })
+    .option('d', {
+      alias: 'include-dev',
+      demandOption: false,
+      default: false,
+      describe: 'Include dev dependencies',
+      type: 'boolean',
+    })
+    .option('v', {
+      alias: 'show-versions',
+      demandOption: false,
+      default: false,
+      describe: 'Show package versions in chart names',
+      type: 'boolean',
+    })
+    .option('p', {
+      alias: 'path',
+      demandOption: false,
+      describe: 'The path to the application to audit',
+      type: 'string',
+    })
+    .option('md', {
+      alias: 'max-depth',
+      demandOption: false,
+      describe: 'Max depth to represent in charts',
+      type: 'number',
+    })
+    .option('ms', {
+      alias: 'min-severity',
+      demandOption: false,
+      describe: 'Min issue severity to represent in charts',
+      type: 'string',
+    })
+    .option('lp', {
+      alias: 'license-policy',
+      demandOption: false,
+      describe: 'Custom license policy JSON string',
+      type: 'string',
+    })
+    .option('f', {
+      alias: 'from',
+      demandOption: false,
+      default: 'registry',
+      describe: 'Load data from "registry" or "disk"',
+      type: 'string',
+    })
+    .option('fo', {
+      alias: 'fail-on',
+      demandOption: false,
+      default: '[]',
+      describe: 'Fail policy JSON string',
+      type: 'string',
+    })
+    .option('s', {
+      alias: 'summary',
+      demandOption: false,
+      default: true,
+      describe: 'Print a summary of the audit results to the console',
+      type: 'boolean',
+    })
+    .option('rs', {
+      alias: 'root-is-shell',
+      demandOption: false,
+      default: false,
+      describe: 'Root project is a shell with a single dependency',
```

Comment:
Maybe we want to be a bit more descriptive here? Something along the lines of "Wrap the project package json in a root package.json with a single dependency. Helps with issues caused by dev dependencies"

Reply:
Hmm maybe it's better if we actually set this to be hidden from the `--help` output? I don't think it can benefit our users much at this point and it's confusing.